index.php

```diff
@@ -14,7 +14,8 @@
         {
             setcookie(REFERING_COOKIENAME, $ID, time() + (14*24*60*60), '', GAMEURL_DOMAIN);
         }
-        header('Location: reg.php');
+        //header('Location: reg.php');         //reg.php not exists - commented
```

Comment:
That commented out line, and the comment below, are not needed. The commit message should state what has been fixed, and that's it - there's no need to leave trace of every single fix in the code itself. That's where the git's log, and git's blame jumps in if someone needs to check the history.

Reply:
Okay, In the last committed file there is only the correct line of code:
https://github.com/mdziekon/UniEngine/pull/38/commits/3978d5e9684ac04d247e89c474f96d05513f9884

Regards,
Alfredo.